README.md

```diff
@@ -0,0 +1,50 @@
+# mumble-web
+
+This role installs and configures [mumble-web](https://github.com/Johni0702/mumble-web).
+The [webrtc version](https://github.com/Johni0702/mumble-web/tree/webrtc) is used.
+
+## Requirements
+
+Debian.
+
+## Role Variables
+
+| Name       | Required/Default | Description                                 |
+| ---------- | :--------------: | ------------------------------------------- |
+| mumble_web |    _required_    | See [Mumble web config](#mumble-web-config) |
+
+## Mumble web config
+The dict mumble_web can have the following entries:
+
+| Name        |   Required/Default    | Description                                                                                                                                                                                                  |
+| ----------- | :-------------------: | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| build_base  |        `/tmp`         | The base folder for building mumble-web                                                                                                                                                                      |
+| deploy_path | `/var/www/mumble-web` | The path to install mumble-web into                                                                                                                                                                          |
+| version     |      _required_       | The sha256 hash of the commit in [mumble-web](https://github.com/Johni0702/mumble-web) to use.                                                                                                               |
```

Comment:
You probably mean `sha1`, don't you?

Reply:
Yes. But actually git is slowly transitioning to sha256. Will just write 
```suggestion
| version     |      _required_       | The hash of the commit in [mumble-web](https://github.com/Johni0702/mumble-web) to use.                                                                                                               |
```